operators/pkg/instance-controller/container_logic.go

```diff
@@ -146,6 +147,26 @@ func buildContainerInstanceDeploymentSpec(
 			// LivenessProbe:   &tigerVncProbe,
 			ReadinessProbe: &tigerVncProbe,
 		},
+		{
+			Name:            "filebrowser",
+			Image:           o.FileMgrImg + ":v2.15.0",
+			Resources:       buildResRequirements(1, 50, resource.MustParse("2G")),
```

Comment:
Are these resources correct? Those seem extremely high to me.

Reply:
Indeed. I have verified the resource consumption on Grafana and I will change them accordingly.